slyd/app/components/annotations-plugin/component.js

```diff
@@ -850,12 +887,13 @@ export default Ember.Component.extend({
         this.set('data', null);
         this.set('mappedElement', null);
         this.set('mappedDOMElement', null);
-        this.set('ignores', []);
+        this.set('pluginState.ignores', []);
```

Comment:
This is resetting the global plugin state. I don't think this needs to be reset and the pluginstate object doesn't need to be updated.